acceptance/analyzer_test.go

```diff
@@ -76,317 +77,494 @@ func TestAnalyzer(t *testing.T) {
-							authRegCacheImage, cacheAuthConfig = buildAuthRegistryImage(
-								t,
-								"some-cache-image-"+h.RandString(10),
-								filepath.Join("testdata", "analyzer", "cache-image"),
+							cacheImage = "some-cache-image-" + h.RandString(10)
+
+							cmd := exec.Command(
+								"docker",
+								"build",
+								"-t", cacheImage,
 								"--build-arg", "fromImage="+containerBaseImage,
 								"--build-arg", "metadata="+metadata,
+								filepath.Join("testdata", "analyzer", "cache-image"),
 							)
+							h.Run(t, cmd)
 						})
 
 						it.After(func() {
-							h.DockerImageRemove(t, authRegCacheImage)
+							h.DockerImageRemove(t, cacheImage)
 						})
 
-						when("registry creds are provided in CNB_REGISTRY_AUTH", func() {
-							it("restores cache metadata", func() {
-								output := h.DockerRunAndCopy(t,
-									containerName,
-									copyDir,
-									"/layers",
-									analyzeImage,
-									h.WithFlags(append(
-										dockerSocketMount,
-										"--network", registryNetwork,
-										"--env", "CNB_REGISTRY_AUTH="+cacheAuthConfig,
-										"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
-									)...),
-									h.WithArgs(
-										ctrPath(analyzerPath),
-										"-daemon",
-										"-cache-image", authRegCacheImage,
-										"some-image",
-									),
+						it("ignores the cache", func() {
+							h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7 does not read from the cache")
+
+							h.DockerRunAndCopy(t,
+								containerName,
+								copyDir,
+								ctrPath("/layers"),
+								analyzeImage,
+								h.WithFlags(append(
+									dockerSocketMount,
+									"--env", "CNB_PLATFORM_API="+apiString,
+								)...),
+								h.WithArgs(
+									ctrPath(analyzerPath),
+									"-daemon",
+									"-cache-image", cacheImage,
+									"some-image",
+								),
+							)
+
+							h.AssertPathDoesNotExist(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.sha"))
+							h.AssertPathDoesNotExist(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.toml"))
+						})
+					})
+
+					when("cache image is in a registry", func() {
+						when("auth registry", func() {
+							var authRegCacheImage, cacheAuthConfig string
+
+							it.Before(func() {
+								metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "cache_image_metadata.json"), platform.CacheMetadata{})
+								authRegCacheImage, cacheAuthConfig = buildAuthRegistryImage(
+									t,
+									"some-cache-image-"+h.RandString(10),
+									filepath.Join("testdata", "analyzer", "cache-image"),
+									"--build-arg", "fromImage="+containerBaseImage,
+									"--build-arg", "metadata="+metadata,
 								)
+							})
 
-								assertLogsAndRestoresCacheMetadata(t, copyDir, output)
+							// Don't attempt to remove the image, as it's stored in the test registry, which is ephemeral.
```

Comment:
It's part of commit 067a590.
We used to have:
```
it.After(func() {
  h.DockerImageRemove(t, authRegCacheImage)
})
```
but as written in the comment, it cause us failures from time to time.
As written in the comment, there is no problem with skipping the image removal because the image is stored in an ephemeral registry.